deploy/concierge/deployment.yaml

```diff
@@ -245,6 +245,10 @@ spec:
           effect: NoSchedule
         - key: node-role.kubernetes.io/control-plane #! The new name for these nodes as of Kubernetes 1.24.
           effect: NoSchedule
+        - key: kubernetes.io/arch
+          effect: NoSchedule
+          operator: Equal
+          value: arm64 #! Also allow running these pods on arm64 nodes.
```

Comment:
Should these deployments explicitly tolerate all supported architectures?

Reply:
(Since I had to look it up, yes `kubernetes.io/arch` is a magic name: https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-arch)

Reply:
Hmm good question. I didn't need it in my experiments, but it seems like it should not hurt.

My GKE cluster automatically tainted the arm64 nodes like this, which will "push away" any pod that does not have the corresponding toleration.
```
    taints:
    - effect: NoSchedule
      key: kubernetes.io/arch
      value: arm64
```

My Kind cluster has no taints on the amd64 nodes, which should allow any pod to schedule there regardless of its tolerations.

I haven't tried a cluster with nodes of both types yet. I assume that they would taint the arm64 nodes and not taint the amd64 nodes, because otherwise all old apps would need to be updated to add tolerations for amd64, but I'm not sure.

Reply:
Seems like it shouldn't hurt, so I added both tolerations (`amd64` and `arm64`) to our deployments.